src/components/security_manager/test/crypto_manager_impl_test.cc

```diff
@@ -118,6 +125,8 @@ class CryptoManagerTest : public testing::Test {
   utils::SharedPtr<CryptoManagerImpl> crypto_manager_;
   utils::SharedPtr<MockCryptoManagerSettings> mock_security_manager_settings_;
   static std::string certificate_data_base64_;
+  std::vector<int> forced_protected_services;
```

Comment:
@ValeriiMalkov 
- named wrongly according to SDL Coding Style

Reply:
@AByzhynar fixed at the commit 0eee356873e4c810e70e48657dc1f4dc1a8467fe